lib/reporters/junit/index.js

```diff
@@ -34,6 +36,9 @@ JunitReporter = function (newman, reporterOptions) {
             accumulator[execution.id].push(execution);
         }, {});
 
+        date = new Date(_.get(newman, 'summary.run.timings.started'));
```

Comment:
`date` is not needed, `timestamp` can be assigned directly.

Reply:
Done.